dashboard/app/helpers/plc/learning_modules_helper.rb

```diff
@@ -2,4 +2,11 @@ module Plc::LearningModulesHelper
   def options_for_plc_task_learning_modules
     Plc::LearningModule.all.pluck(:name, :id).sort
   end
+
+  def options_for_evaluation_answer_modules(course_unit)
+    course_unit.plc_learning_modules.order(:required, :name).map do |learning_module|
+      # Boo Ruby, you can't add nil to a string without choking?
+      [learning_module.name += (' - Required' if learning_module.required).to_s, learning_module.id]
```

Comment:
I don't think you want `+=` here — you're modifying the actual learning_module name, but it doesn't persist since `learning_module.save` is never called.  Prefer:
```ruby
["#{learning_module.name}#{' - Required' if learning_module.required}", learning_module.id]
```